src/tests/aes.rs

```diff
@@ -874,6 +880,13 @@ fn test_aes_macs() {
             )
         );
 
+        /* test that we can get correct indicators based on inputs */
```

Comment:
Only nit here is that I think this line should say why the fips indicator is expected to be false here